src/applications/vaos/utils/constants.js

```diff
@@ -176,9 +176,9 @@ export const FACILITY_TYPES = {
 };
 
 export const FACILITY_SORT_METHODS = {
-  DISTANCE_FROM_RESIDENTIAL: 'distanceFromResidential',
-  DISTANCE_FROM_CURRENT_LOCATION: 'distanceFromCurrentLocation',
-  ALPHABETICAL: 'alphabetical',
+  distanceFromResidential: 'distanceFromResidentialAddress',
+  distanceFromCurrentLocation: 'distanceFromCurrentLocation',
+  alphabetical: 'alphabetical',
```

Comment:
Renamed the objects within this constant to camel case to follow pattern of other constants